tests/test_manager.py

```diff
@@ -33,14 +34,17 @@
 from tests.conftest import MY_FIRST_AEA_PUBLIC_ID
 
 
-class TestMultiAgentManagerAsyncMode:  # pylint: disable=unused-argument,protected-access,attribute-defined-outside-init
+@patch("aea.aea_builder.AEABuilder.install_pypi_dependencies")
+class TestMultiAgentManagerAsyncMode(
+    TestCase
```

Comment:
What's the benefit of also using TestCase now...?

Reply:
got some issues with teardown/setup so decided to use TestCase as a bse class, actually it's a proper test case definition.